zcash_primitives/src/note_encryption.rs

```diff
@@ -1347,8 +1367,13 @@ mod tests {
             let ock = prf_ock(&ovk, &cv, &cmu, &epk);
             assert_eq!(ock.as_bytes(), tv.ock);
 
+            //let to = PaymentAddress {
+            //    pk_d,
+            //    diversifier: Diversifier(tv.default_d),
+            //};
+            //TODO: was different in master
             let to = PaymentAddress::from_parts(Diversifier(tv.default_d), pk_d).unwrap();
-            let note = to.create_note(tv.v, rcm, &JUBJUB).unwrap();
+            let note = to.create_note(AssetType::Zcash, tv.v, rcm, &JUBJUB).unwrap();
```

Comment:
Should there be tests for other asset types as well?

Reply:
Yeah, I'll add that to the list of remaining tasks